indico/queries/documents.py

```diff
@@ -5,27 +5,7 @@
 
 from indico.client.request import RequestChain, GraphQLRequest, HTTPMethod, HTTPRequest
 from indico.types.jobs import Job
-
-
-class _UploadDocument(HTTPRequest):
```

Comment:
This seems redundant since the identical code is in storage.py.

Reply:
Yes forgot to move it to that class after making a common one.